models/orders.model.js

```diff
@@ -1,58 +1,34 @@
-import database from '../database.js'
+import { Order } from '../db/models/Order.js'
 
 export default {
   addOrder: async (customerId, clockId, masterId, cityId, startTime, endTime) => {
-    const order = await database.query(
-      `
-            INSERT INTO orders ("customerId","clockId","masterId","cityId","startTime","endTime") 
-            VALUES ($1,$2,$3,$4,$5,$6) RETURNING *`,
-      [customerId, clockId, masterId, cityId, startTime, endTime]
-    )
-    return order.rows[0]
+    const order = await Order.create({ customerId, clockId, masterId, cityId, startTime, endTime })
+    return order
   },
   getOrders: async () => {
-    const orders = await database.query(`
-            SELECT orders.id, customers.name, customers.email, clocks.size, 
-            clocks."timeToFix", masters.name AS master, cities.name AS city,
-            "startTime", "endTime"
-            FROM orders 
-            INNER JOIN customers ON "customerId" = customers.id 
-            INNER JOIN clocks ON "clockId" = clocks.id 
-            INNER JOIN masters ON "masterId" = masters.id 
-            INNER JOIN cities ON "cityId" = cities.id
-            ORDER BY id DESC`)
-    return orders.rows
-  },
-  getOrdersList: async () => {
-    const orders = await database.query(`SELECT id, "masterId", "cityId",
-            "startTime", "endTime"
-            FROM orders `)
-    return orders.rows
+    const orders = await Order.findAll({
+      order: [['id', 'DESC']],
+      include: ['city', 'master', 'customer', 'clock']
```

Comment:
Try to use include with DBModel, without array of string
https://sequelize.org/docs/v6/advanced-association-concepts/eager-loading/#fetching-a-single-associated-element